lesson_09/test_homework_09.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+from homework_09 import (
+    multiplication_table,
+    fibonacci_generator_solution,
+    is_palindrome,
+    find_max_length_palindrome,
+)
+
+
+class TestSomeFunctions(unittest.TestCase):
+
+    def test_multiplication_table_positive(self):
+        """ Multiplication positive test. Positive integer number as function argument. """
+        actual_result = multiplication_table(5)
+        expected_result = [5, 10, 15, 20, 25]
+        self.assertEqual(actual_result.pop(), expected_result.pop())
+        self.assertEqual(len(actual_result), len(actual_result))
```

Comment:
```suggestion
        self.assertEqual(actual_result, expected_result)
```
тут очікується повне співпадіння. Знову таки добре, що ви погралися, але однієї умови досить.

Reply:
Зроблено.